src/utils/StatUtil.js

```diff
@@ -33,7 +33,13 @@ export class StatUtil {
     this.log('bgPrefs', { bgBounds: this.bgBounds, bgUnits: this.bgUnits });
   };
 
+  filterCBGDataByDefaultSampleInterval = () => {
```

Comment:
All generated stats will use the default sample interval range for calculations, even if the user is viewing 1min data